src/foam/nanos/notification/email/EmailTemplateEngine.js

```diff
@@ -59,8 +59,7 @@ foam.CLASS({
         Action simpleValAction = new Action() {
           @Override
           public Object execute(Object val, ParserContext x) {
-            Object[] valArr  = (Object[]) val;
-            Object[] val0    = (Object[]) valArr[0];
+            Object[] val0    = (Object[]) val;
```

Comment:
Should this file be in this PR?

Reply:
it depends in the change in Until parser. So should be submitted together